### ISSUE TITLE
Can't specify a keypath separator at the same time as creating a dictionary from yaml

### ISSUE THREAD

**Issue by jerkmcjerkface** [fabiocaccamo/python-benedict#12]
**Python version**
2.7

**Package version**
python-benedict                    0.14.1

**Current behavior (bug description)**
When attempting to create a dictionary from a yaml file, if will fail if the data inside the yaml file contains a "." .  This is expected since the period is the default separator.  However, this seems like it should make it work, but it doesn't:

` d = benedict.from_yaml("test.yml", keypath_separator=None)`

When attempting to create a dictionary from a yaml file, the option to set the keypath (or disable it does not work) does not work.  

test.yml contents:
```
192.168.0.1:
  test: value
  test2: value2
value:
  value_with_period: 12.34.45

```

**Expected behavior**

This should work, if the test.yml contains a value that contains the separator.
` d = benedict.from_yaml("test.yml", keypath_separator=None)`

Is there a better way to do this? I see similiar issues, but I don't see something that is specifically helpful.

**Comment by fabiocaccamo** [comment]
@jerkmcjerkface thank you for reporting this issue. I will add `keypath_separator` support to all `from_{format}` methods as soon as possible.

In the meanwhile you can do the same in this way:
```python
d = benedict('test.yml', format='yaml', keypath_separator=None)
```

**Comment by fabiocaccamo** [comment]
Fixed in `0.18.1` version, enjoy :octocat:
